pkg/mounter/oss/oss_fuse_manager.go

```diff
@@ -86,4 +87,7 @@ type Options struct {
 	AuthType      string `json:"authType"`
 	FuseType      string `json:"fuseType"`
 	ReadOnly      bool   `json:"readOnly"`
+
+	// pod template
+	DnsPolicy corev1.DNSPolicy `json:"dnsPolicy"`
```

Comment:
I suggest exposing more abstract configuration options to users, as they may not be aware of our network policy being hostNetwork. A focus on coreDNS and nodeDNS settings would suffice.